crates/ui/src/slider.rs

```diff
@@ -168,6 +187,23 @@ impl SliderState {
         self
     }
 
+    /// Set the scale of the slider, default: Linear
+    pub fn scale(mut self, scale: SliderScale) -> Self {
```

Comment:
Please also add this option to `SliderStory` for example.

Reply:
Added an example to the story where a logarithmic slider would be the right choice

https://github.com/user-attachments/assets/33d7cfb5-bb4a-43a9-a11d-c13cf8fe5de7